arcgis-ios-sdk-samples/Layers/ArcGIS vector tiled layer (custom style)/VectorTileCustomStyleViewController.swift

```diff
@@ -17,40 +17,89 @@ import UIKit
 import ArcGIS
 
 class VectorTileCustomStyleViewController: UIViewController, VectorStylesVCDelegate {
-    @IBOutlet var mapView: AGSMapView!
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            mapView.map = AGSMap()
+        }
+    }
     
-    private var itemIDs = ["1349bfa0ed08485d8a92c442a3850b06",
+    private let itemIDs = ["1349bfa0ed08485d8a92c442a3850b06",
                            "bd8ac41667014d98b933e97713ba8377",
                            "02f85ec376084c508b9c8e5a311724fa",
-                           "1bf0cc4a4380468fbbff107e100f65a5"]
-    private var shownItemID: String? {
-        return ((mapView.map?.basemap.baseLayers.firstObject as? AGSArcGISVectorTiledLayer)?.item as? AGSPortalItem)?.itemID
+                           "1bf0cc4a4380468fbbff107e100f65a5",
+                           "2056bf1b350244d69c78e4f84d1ba215"]
+    
+    let temporaryURL: URL? = {
```

Comment:
```suggestion
    let temporaryURL: URL = {
```

1. This doesn't need to be optional

2. Since this is only used by the load function, consider making it a local constant.